### DISCUSSION TITLE
Map on Photoprism

### DISCUSSION THREAD

**Discussion by EgalitarianMonkey** [Radiokot/photoprism-android-client#164] [Ideas]
Hello, 
I'm very interested by the map feature on Photoprism. Do you plan to add it in the future? 
Sorry if this was already discussed.
Otherwise great job, 
Thanks.

**Comment by Radiokot** [comment]
Hi, I'd like to add it

**Reply by EgalitarianMonkey** [reply]
Can you tell if you can make it in the next months?

**Reply by Radiokot** [reply]
Yes

**Reply by EgalitarianMonkey** [reply]
That's good. Thanks.
Can you tell a realistic month as a target for the feature?

**Reply by Radiokot** [reply]
I think so. Please, consider [supporting the development](https://radiokot.com.ua/tip).

**Comment by Radiokot** [comment]
Hi there 👋🏻 
There's now a map in the gallery: https://github.com/Radiokot/photoprism-android-client/releases/tag/1.42.0(63)
